frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -213,7 +213,7 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
                                                 </div>
                                             )}
                                             {survey.type !== SurveyType.API ? (
-                                                <div className="mt-6" style={{ maxWidth: 320 }}>
+                                                <div className="mt-6">
```

Comment:
I _think_ same width as popup was required for something, although I don't remember what. @jurajmajerik does.

Reply:
yep, the dropdown overflows in case of a long question:

![image](https://github.com/PostHog/posthog/assets/22996112/a304d71f-47dd-43c1-8c20-280af70efee4)

Reply:
Ah great catch. Fixed in the latest commit. We're trying to move away from inline styles
<img width="772" alt="Screenshot 2023-10-31 at 13 39 10" src="https://github.com/PostHog/posthog/assets/6685876/6f94fb21-9a93-4b38-b230-ac5d8bc76586">

Reply:
Yeah, this is not great. We use Tailwind with rem's everywhere, but survey popups are styled with pure CSS (I'm not quite sure why, maybe @liyiy knows). By using inline styles here, we avoided creating a whole SCSS file just to style this.

Actually, now that I'm thinking about it, it's possible to set a `w-{width}` custom class in Tailwind, I'll note it down for my next round of housekeeping :)

Reply:
It was originally styled with pure CSS to make it easier to reflect the actual survey popup's code. However if we can get matching visual tests for both of them so that they remain in sync some other way, that's a better improvement, though I don't think CSS files in general are that bad? Unless we're trying to get rid of all of them in posthog-3000...